android/guava/src/com/google/common/hash/Hashing.java

```diff
@@ -281,6 +281,10 @@ private static class Sha512Holder {
    * Returns a hash function implementing the Message Authentication Code (MAC) algorithm, using the
    * MD5 (128 hash bits) hash function and the given secret key.
    *
+   * <p>If you are designing a new system that needs HMAC, prefer {@link #hmacSha256} or other
+   * future-proof algorithsm <a
```

Comment:
@cpovirk, there is a small typo here "algorith<del>s</del>m<ins>s</ins>":
```suggestion
   * future-proof algorithms <a
```

(same also in the other changed lines)

Reply:
Thanks! We should be able to fix that. :-)